traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/ds/SteeringWatcher.java

```diff
@@ -22,7 +22,7 @@
 	private static final Logger LOGGER = Logger.getLogger(SteeringWatcher.class);
 	private SteeringRegistry steeringRegistry;
 
-	public static final String DEFAULT_STEERING_DATA_URL = "https://${toHostname}/internal/api/1.3/steering.json";
+	public static final String DEFAULT_STEERING_DATA_URL = "https://${toHostname}/api/2.0/steering.json";
```

Comment:
I'm pretty sure these changes won't work because 2.0 doesn't support `.json` suffixes.

Also, I believe this directory structure will need updated to reflect the new 2.0 route as well:
```
./core/target/test-classes/internal
./core/target/test-classes/internal/api
./core/target/test-classes/internal/api/1.3
./core/target/test-classes/internal/api/1.3/steering2.json
./core/target/test-classes/internal/api/1.3/federations.json
./core/target/test-classes/internal/api/1.3/steering.json
```

Reply:
The test resources paths have changed and as a result it shouldn't generate the `./core/target/test-classes/internal` folder anymore.

Reply:
Ah, I see the name changes in the PR now, so that leaves the `.json` issue.

Reply:
Should be resolved now